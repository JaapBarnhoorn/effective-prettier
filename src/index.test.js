import { promises as fs } from "fs"

import { formatText } from "."

describe("Auto formatting", () => {
  test("Autofix 1", async () => {
    const filePath = "./src/fixtures/autofix1.js"
    const input = await fs.readFile(filePath, { encoding: "utf-8" })
    const output = await formatText(input, { filePath, verbose: true })
    expect(output).toMatchSnapshot()
  })
})
