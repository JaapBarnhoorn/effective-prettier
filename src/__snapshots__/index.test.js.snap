// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auto formatting Autofix 1 1`] = `
"function hello() {
  return \\"powerful\\"
}
"
`;
